### PR TITLE
Fix multiple pairs config

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: octobot_evaluators/evaluators/evaluator_factory.py [hunk 1]
```diff
@@ -167,3 +158,31 @@ async def create_all_type_evaluators(tentacles_setup_config: object,
...
+    import octobot_trading.api as exchange_api
+    exchange_id = exchange_api.get_exchange_id_from_matrix_id(exchange_name, matrix_id)
+    exchange_manager = exchange_api.get_exchange_manager_from_exchange_name_and_id(exchange_name, exchange_id)
+    for name, symbol_list in symbols_by_crypto_currencies.items():
+        if symbol_list:
+            # in case pairs are listed by reference market in use config, iterate over each pair
+            for symbol in symbol_list:
+                base = exchange_api.get_base_currency(exchange_manager, symbol)
+                crypto_currency_name_by_crypto_currencies[base] = \
+                    crypto_currency_name_by_crypto_currencies.get(base, name)
```
Comment (Herklos) [inline octobot_evaluators/evaluators/evaluator_factory.py:175]: It was `crypto_currency_name_by_crypto_currencies.get(base, "") + name` and now `crypto_currency_name_by_crypto_currencies.get(base, name)` ?

File: octobot_evaluators/evaluators/evaluator_factory.pxd [hunk 2]
```diff
@@ -26,3 +26,5 @@ cdef list _get_symbols_to_create(object evaluator_class,
                                  list symbols)
 cdef list _get_time_frames_to_create(object evaluator_class,
                                      list time_frames)
+cdef tuple _extract_traded_pairs(dict symbols_by_crypto_currencies, str exchange_name, str matrix_id)
+cdef list _filter_pairs(list pairs, str required_ticker, object exchange_api, object exchange_manager)
```
Comment (Herklos) [inline octobot_evaluators/evaluators/evaluator_factory.pxd:30]: It's returning a set isn't it ?

File: octobot_evaluators/evaluators/evaluator_factory.py (hunk 1 above)
Comment (GuillaumeDSM) [reply octobot_evaluators/evaluators/evaluator_factory.py:175]: yep, it was an issue

Comment (Herklos) [reply octobot_evaluators/evaluators/evaluator_factory.py:175]: :+1:

**Review by Herklos — APPROVED** [review]

**Pull request merged**